front/lib/api/oauth.ts

```diff
@@ -216,6 +216,25 @@ const PROVIDER_STRATEGIES: Record<
       return getStringFromQuery(query, "state");
     },
   },
+  zendesk: {
+    setupUri: (connection) => {
+      const scopes = ["tickets:read", "hc:read"];
```

Comment:
Shouldn't those scopes be aligned with the one in zendesk.rs?

Reply:
Indeed - mb in zendesk.rs, I reverted a change. Adding the fix in this PR